ghproxy/ghproxy.go

```diff
@@ -101,6 +109,7 @@ func flagOptions() *options {
 	flag.IntVar(&o.maxConcurrency, "concurrency", 25, "Maximum number of concurrent in-flight requests to GitHub.")
 	flag.StringVar(&o.pushGateway, "push-gateway", "", "If specified, push prometheus metrics to this endpoint.")
 	flag.DurationVar(&o.pushGatewayInterval, "push-gateway-interval", time.Minute, "Interval at which prometheus metrics are pushed.")
+	flag.StringVar(&o.logLevel, "log-level", "debug", "Log level is one of [debug|info|warn|error|fatal].")
```

Comment:
IIRC there's a helper in logrus to list the levels possible

Reply:
Done.